web/src/app/listing/component.html

```diff
@@ -42,7 +42,7 @@
 			<span *ngIf="showRemoveButton" (click)="removeButtonClick()" class="remove-button position-absolute"  [placement]="['right','bottom','left','top','auto']" ngbPopover="Click on this button to remove class from sidebar" container="body" triggers="mouseenter:mouseleave">
 				<img src="assets/images/remove.svg" />
 			</span>
-			<div class="menu position-absolute" (click)="showingMenu=!showingMenu">
+			<div class="menu position-absolute" (click)="showingMenu=!showingMenu" (click)="showingDescription=!showingDescription">
```

Comment:
I think you should create an expand / toggle function to handle this rather than have two click handlers